### PR TITLE
Add json parser exercise

### PR DESCRIPTION
**Pull request by HamedNourani**

### CODE REVIEW

**Comment by URAmiRBin** [comment]
Ill review this

**Review by URAmiRBin — COMMENTED** [review]
Only add the csharp files

File: Algorithms and Data Structures/JasonConvertExercise/JasonConvertExercise/PlayerData.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿namespace JasonParserExercise
```
Comment (URAmiRBin) [inline Algorithms and Data Structures/JasonConvertExercise/JasonConvertExercise/PlayerData.cs]: the namespace name should be `JsonParser`

File: Algorithms and Data Structures/JasonConvertExercise/JasonConvertExercise/Program.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+                if (convertedPlayerData.username == null)
+                {
+                    Console.WriteLine("username is a necessary field and cannot be null!");
+                    Parse();
+                }
+                else if (convertedPlayerData.level == 0)
+                {
+                    convertedPlayerData.level = 1;
+                }
```
Comment (URAmiRBin) [inline Algorithms and Data Structures/JasonConvertExercise/JasonConvertExercise/Program.cs]: the default values should be handled better, think about we want to have multiple default values for each field, multiple ifs is code smell and should indicate that you have to find a better way to solve this problem

File: Algorithms and Data Structures/JasonConvertExercise/JasonConvertExercise/PlayerData.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+{
+    public class PlayerData
+    {
+        public string username;
+        public int level;
```
Comment (URAmiRBin) [inline Algorithms and Data Structures/JasonConvertExercise/JasonConvertExercise/PlayerData.cs]: the exercise says this: 

> Create a C# class named PlayerData that can represent the player's information, including fields for username, level, guild, items, and skinId. The class should also include appropriate properties and constructors.

where are the other properties?

**Review by URAmiRBin — CHANGES_REQUESTED** [review]

**Comment by HamedNourani** [comment]
@URAmiRBin 
I resolved the issues you had requested.

**Review by URAmiRBin — APPROVED** [review]

**Pull request merged**